### PR TITLE
adds ocelots m41a, deletes a few unused sprites

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
adds ocelots m41a, and relores it a bit I guess, I guess the balance is a bit changed since the sprite is supposed to have the grenade launcher no matter what so its irremoveable, so I added mag harness in return? You can't get this gun anyway.
I removed some unused gun sprites that I forgot to remove.
I also added an invisible sprite to marine-weapons as otherwise " " setup made it show up as the last icon in the file.
image

## Why It's Good For The Game
I hate the old sprites. New sprites good.
Clean up good.
Good.

## Changelog
:cl:
del: Removed a few old sprites. M16a4, m41a, etc
balance: m41a gl is unremoveable, has mag harness now
tweak: m41a relored, is now named HK-11
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -221. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -376,11 +376,12 @@
 
 
 //-------------------------------------------------------
-//M41A TRUE AND ORIGINAL
+//M41A
```
Comment (SplinterGP) [inline code/modules/projectiles/guns/rifles.dm:379]:
```suggestion
//HK-11
```

**Review by SplinterGP — APPROVED** [review]
I like it, as much as it goes if we make WO a proper gamemode id love that gun for it. To give it a use

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-229). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**